global.json

```
@@ -5,10 +5,7 @@
   "tools": {
     "dotnet": "5.0.100-alpha1-013788",
     "runtimes": {
-      "dotnet/x64": [
-        "$(MicrosoftNETCoreAppRuntimeVersion)"
-      ],
-      "dotnet/x86": [
+      "dotnet": [
         "$(MicrosoftNETCoreAppRuntimeVersion)"
       ]
```

```suggestion
      ],
      "dotnet/x86": [
        "$(MicrosoftNETCoreAppRuntimeVersion)"
      ]
```
If I'm understanding things correctly, `dotnet-install.sh` and `dotnet-install.ps1` will install the runtime for the current architecture. Then, `dotnet-install.sh` will ignore the x86 request while `dotnet-install.ps1` will install that runtime in the .dotnet/x86/ folder.

Basically, this limits your change to `"dotnet/x64"` ➡️ `"dotnet"`.

---

Thanks, that's a **great** idea!